data/categories.json

```
@@ -3,90 +3,105 @@
     "_id": {
       "$oid": "6462a6cd4c3d0ddd28897f8a"
     },
-    "name": "Seafood"
+    "name": "Seafood",
+    "url": "/categories/Seafood.jpg"
   },
   {
     "_id": {
       "$oid": "6462a6cd4c3d0ddd28897f8b"
     },
-    "name": "Lamb"
+    "name": "Lamb",
+    "url": "/categories/Lamb.jpg"
   },
   {
     "_id": {
       "$oid": "6462a6cd4c3d0ddd28897f8c"
     },
-    "name": "Starter"
+    "name": "Starter",
+    "url": "/categories/Starter.jpg"
   },
   {
     "_id": {
       "$oid": "6462a6cd4c3d0ddd28897f8d"
     },
-    "name": "Chicken"
+    "name": "Chicken",
+    "url": "/categories/Chicken.jpg"
   },
   {
     "_id": {
       "$oid": "6462a6cd4c3d0ddd28897f8e"
     },
-    "name": "Beef"
+    "name": "Beef",
+    "url": "/categories/Beef.jpg"
   },
   {
     "_id": {
       "$oid": "6462a6cd4c3d0ddd28897f8f"
     },
-    "name": "Dessert"
+    "name": "Dessert",
+    "url": "/categories/Dessert.jpg"
```

```suggestion
    "url": "/categories/Desserts.jpg"
```
Або краще навіть перейменувати файл у "Dessert.jpg", щоб усі файли були в однині.